libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -1589,11 +1589,16 @@ - (void)notifyUserChange:(NSString *)notificationName withUser:(SFUserAccount *)
 
 - (void)notifyUserCancelledOrDismissedAuth:(SFOAuthCredentials *)credentials andAuthInfo:(SFOAuthInfo *)info
  {
-    NSDictionary *userInfo = @{ kSFNotificationUserInfoCredentialsKey:credentials,
-                                kSFNotificationUserInfoAuthTypeKey: info };
+    NSMutableDictionary *userInfo = [NSMutableDictionary new];
+    userInfo[kSFNotificationUserInfoCredentialsKey] = credentials;
+    if (info) {
```

Comment:
But why is the info null on the second cancel?

Reply:
It's set to nil [here](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/c7c22f1d883f036a4beb81173b0347f7c4ae2b45/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m#L361) after the first cancel call and then would be populated again after the refresh but the second cancel can happen before the refresh finishes

Reply:
👍